Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,50 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the **users** group (which has a GID of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u`, `--uid` _uid_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g`, `--gid` _gid_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s`, `--shell` _shell_: The shell binary for this login. The default is `/bin/Shell`. 
+* `-m`, `--create-home`: Create a home directory for this new user at `/home/username`.
```

Comment:
It's created at whatever path you specify with `--home-dir` (and with `/home/username` as fallback). And you need to indicate in some way that `username` is not literally `username`, but whatever username is specified.

Reply:
And it's actually called `login`, not `username`, above?